README.md

```diff
@@ -1 +1,66 @@
-# first-capstone-summit
\ No newline at end of file
+# html-css-js-capstone-project
+
+This is my final project for HTML & CSS MODULE from the microverse curriculum
+
+<p align="center">
+  <img src="./img/mobile.png"/>
+</p>
+
+<hr>
+
+<p align="center">
+  <img src="./img/desktop.png"/>
+</p>
+
+This project is meant to test student ability's on being able to put together all that they have learnt in a module to ensure they mastered it well.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+Here's a [video ](https://www.loom.com/share/f4a8530a7a7c4fc28313a20d7f0542db) of me giving a few description of the project.
```

Comment:
#### OPTIONAL

- [ ] Kindly be more audible or check your mic settings when recording videos 🤗